src/store/flow/flow.ts

```diff
@@ -16,20 +17,69 @@ import moment from 'moment'
 import { ActionTree, GetterTree, MutationTree } from 'vuex'
 import { IRootState } from '@/store'
 import {
-  defaults, includes, forEach, cloneDeep, get, has,
+  defaults, 
+  includes, 
+  forEach, 
+  cloneDeep, 
+  get, 
+  has, 
+  omit
 } from 'lodash'
 import { discoverContentTypesFor } from '@/store/flow/resource'
 import { computeBlockPositionsFrom } from '@/store/builder'
 import { IFlowsState } from '.'
 
 export const getters: GetterTree<IFlowsState, IRootState> = {
-  activeFlow: (state) => state.flows.length && getActiveFlowFrom(state as unknown as IContext),
-
+  //We allow for an attempt to get a flow which doesn't yet exist in the state - e.g. the firstFlowId doesn't correspond to a flow
+  activeFlow: (state) => {
+    if(state.flows.length) { 
+      try {
+        return getActiveFlowFrom(state as unknown as IContext)
+      } catch(err) {
+        return 0
+      }
+    }
+  },
+  //TODO - is the IContext equivalent to the Flow Container? Can we say that it should be?
```

Comment:
In your checkbox you have "update activeFlowContainer to match IContainer" but what I'm taking from this is that this isn't something we can do quite yet..? It also might not be easy in this PR if we want to avoid conflicts with Safydy's flow-runner upgrade PR perhaps?

Reply:
My suggestion was to update the set of properties listed below to match the properties on flow runner's `IContainer` to pave way for when we do have access to `IContainer`.

If you're suggesting that this affects more than just a couple places, then yes let's leave it for now!

Reply:
Interesting. I was basing this on the spec. I'll take a quick look at the differences from IContainer

Reply:
Hmm. I can't find IContainer. Where is this..? Not seeing it in the latest flow-runner.